tox.ini

```diff
@@ -11,4 +11,5 @@ deps =
     docker-py
     molecule
 commands =
-    molecule test --all
+    molecule --debug converge
+# molecule --debug test --all
```

Comment:
This would be fine-- personally I find --debug to be too verbose to be helpful, but I'd be fine with this. Just not `converge` only.